### ISSUE TITLE
DOCS: GLCM example hard to follow

### ISSUE THREAD

**Issue by michaelaye** [scikit-image/scikit-image#2469]
In the GLCM Texture Features example, I find the following sentence popping out by not being understandable:

> For each patch, a GLCM with a horizontal offset of 5 is computed.

What is unclear:
* What means 'horizontal'? 
* Offset from what?

I'm looking at this page:
http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_glcm.html#sphx-glr-auto-examples-features-detection-plot-glcm-py

**Comment by sciunto** [comment]
What do you mean by popping out? If I run the script, nothing is printed (skimage 0.12 or dev).

**Comment by soupault** [comment]
Thanks @michaelaye ! It's is indeed hard to understand without looking at the example code and the API docs.
What is meant, I think, that GLCM is called with `distance=[5]` and `angles=[0]`.

Feel free to sumbit a PR with an enhancement.

**Comment by michaelaye** [comment]
I don't know what a distance and angles stand for in GLCM either. Sorry, I can't submit a PR as I don't understand what is going on. ;) Maybe i should just read a primer on GLCMs...
